drinkwater/script.js

```diff
@@ -0,0 +1,54 @@
+
+const glasses = document.querySelector('.glasses.grid');
+const nbOfGlasses = glasses.children.length;
+
+class GlassContainer{
```

Comment:
can you pull the GlasContainer out into a separate file?

Reply:
Okay, no problem, and I will also make the Drinker feature available into website's links

Reply:
Okay, no problem, and I will also make the Drinker feature available in the website's links